BENCH

```diff
@@ -0,0 +1,24 @@
+FROM alpine:3.15
+
+# go path
+ENV PATH=$PATH:/usr/local/go/bin
+
+# rust path
+ENV PATH=$PATH:/root/.cargo/bin
+
+WORKDIR /app
+
+RUN apk add alpine-sdk --update
+
+# enable rust
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly-2021-11-17
+
+# enable go
+COPY --from=golang:1.16-alpine /usr/local/go/ /usr/local/go/
+
+COPY . .
+
+RUN RUSTFLAGS='-C target-cpu=native' cargo bench --no-run
+
+CMD RUSTFLAGS='-C target-cpu=native' cargo bench --bench evm_prover &&\
+    RUSTFLAGS='-C target-cpu=native' cargo bench --bench state_prover
```

Comment:
Is there a way on which we can split these three things in separate docker actions? 
Instead of always running them all? We can always run them all anyway if we want. But at least we can be more selective.

Reply:
I modified in order to indicate `bench name` as following.
```
$ docker-compose run -e BENCH_NAME=state_prover bench
```